dashboard/app/models/sections/section.rb

```diff
@@ -129,27 +132,30 @@ def students_attributes=(params)
 
   # Adds the student to the section, restoring a previous enrollment to do so if possible.
   # @param student [User] The student to enroll in this section.
-  # @return [Follower] The newly created follower enrollment.
+  # @return [ADD_STUDENT_EXISTS | ADD_STUDENT_SUCCESS] Whether the student was already
+  #   in the section or has now been added.
```

Comment:
In changing (and depending on) the return value of this method, we should also test the return values in `section_test.rb`.